src/components/opportunities/jobs/profileCard.tsx

```diff
@@ -0,0 +1,173 @@
+import * as React from 'react';
+import { useSelector } from 'react-redux';
+
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from '@/components/ui/dialog';
+import { Button } from '@/components/ui/button';
+import { Input } from '@/components/ui/input';
+import { Label } from '@/components/ui/label';
+import { toast } from '@/components/ui/use-toast';
+import { axiosInstance } from '@/lib/axiosinstance';
+import { RootState } from '@/lib/store';
+
+interface ProfileProps {
+  profile: {
+    domain?: string;
+    freelancersRequired?: string;
+    skills?: string[];
+    experience?: number;
+    minConnect?: number;
+    rate?: number;
+    description?: string;
+  };
+  projectId: string;
+}
+
+const ProfileCard: React.FC<ProfileProps> = ({ profile, projectId }) => {
+  const [amount, setAmount] = React.useState('');
+  const [descriptionValue, setDescription] = React.useState('');
+  const [dialogOpen, setDialogOpen] = React.useState(false);
+  const user = useSelector((state: RootState) => state.user);
+  const [exist, setExist] = React.useState(false);
+  const [showMore, setShowMore] = React.useState<boolean>(false);
+
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+
+    try {
+      await axiosInstance.post(`/bid`, {
```

Comment:
check if your post request was successful or not and add some toast or error message to reflect if post was successful or filed